### ISSUE TITLE
Support vite-node (part of Vitest/Nuxt 3)

### ISSUE THREAD

**Issue by birkskyum** [denoland/deno#20859]
vite-node - [GitHub](https://github.com/vitest-dev/vitest/tree/main/packages/vite-node)

**Platform**

Darwin 23.0.0 arm64 arm

**Version**

deno 1.37.1

**Repro**

- Make a `test.ts` with the content: `console.log("Hello world!")`
- `deno run -A npm:vite-node ./test.ts`

**Expected**

`Hello world!`

it should work like this:
- `bunx vite-node ./test.ts`

**Actual**

```sh
Hello world!
error: Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at <anonymous> (ext:deno_web/01_dom_exception.js:126:14)
```

**Additional information**

**Comment by bartlomieju** [comment]
This appears to work on `main` - so should be fixed by v1.37.2 going releasing this week.

**Comment by birkskyum** [comment]
Verified that it works for me with Deno 1.37.2. Thanks!
